roles/ec2/tasks/main.yml

```diff
@@ -15,6 +15,10 @@
     instance_tags:
       Name: "{{ redis_name }}"
       Project: "1220828"
+    volumes:
+      - volume_size: 8
+        volume_type: gp2
+        device_name: /dev/sda1
   tags:
```

Comment:
Why add a volume now? Didn't this work before?
Shouldn't you add the device_type and delete_on_termination here as well?